libs/isograph-react/src/core/check.ts

```diff
@@ -10,6 +10,7 @@ import {
 import { logMessage } from './logging';
 
 export type ShouldFetch = 'Yes' | 'No' | 'IfNecessary';
+export type RequiredShouldFetch = 'Yes' | 'No';
```

Comment:
nit: might be slightly clearer to have `ShouldFetch = RequiredShouldFetch | 'IfNecessary'`